io/router/router.go

```diff
@@ -182,6 +182,8 @@ func (q *Router) Queue(events ...event.Event) bool {
 			}
 		case clipboard.Event:
 			q.cqueue.Push(e, &q.handlers)
+		case transfer.DataEvent:
+			q.handlers.Add(nil, e)
```

Comment:
This just delivers the even to the `nil` tag. I had expected the event be delivered to the current target, similar to https://git.sr.ht/~eliasnaur/gio/tree/main/item/io/router/pointer.go#L859. See also the `widget/dnd_test.go` for an end-to-end use of the transfer protocol.